src/Storages/MergeTree/MergeTreeIndexConditionText.cpp

```diff
@@ -381,6 +366,31 @@ bool traverseArrayFunctionNode(const RPNBuilderTreeNode & index_column_node, con
     }
     return false;
 }
+
+}
+
+std::vector<String> MergeTreeIndexConditionText::stringToTokens(const Field & field) const
+{
+    std::vector<String> tokens;
+    const String &value = field.safeGet<String>();
```

Comment:
Minor: `const String & value`

Reply:
The `&` will disappear with the preprocessor because we need a temporal "pre-processed" temporal string. So, don't bother with this